test.txt

```diff
@@ -1 +1,2 @@
-placeholder
+AWS_ACCESS_KEY_ID:  ********************
+AWS_SECRET_ACCESS_KEY:  0+rfumaQJh8p+ViUGaBTeCgimS2ukgGnKrLQICgH
```

Comment:
:exclamation:**Cycode**: Secret of type: 'Aws Secret Access Key' was found.
Severity: High

Would you like to exclude this secret from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_secret_false_positive** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_ignore_everywhere** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_ignore_here** | Applies to this request only |
| **#cycode_secret_revoked** | Applies to this secret value for all repos in your organization |

 <!-- c099890b-92d5-4159-b2de-c5a8d906096e -->

Reply:
#cycode_secret_false_positive

Reply:
:+1: Cycode has finished processing your request. Please review your status checks.